Tests/Invoke-DockerTag.Tests.ps1

```diff
@@ -24,91 +24,91 @@ Describe 'Tag docker images' {
 
     Context 'Docker tags public registry images' {
 
-        It 'tags docker image with image name only' {
-            Invoke-DockerTag -SourceImage 'lalaland' -TargetImage 'lololand'
+        It 'tags image by image name with image name' {
+            Invoke-DockerTag -ImageName 'oldname' -NewImageName 'newimage'
             $result = GetMockValue -Key "mock"
             Write-Debug $result
-            $result | Should -BeLikeExactly "docker tag lalaland:latest lololand:latest"
+            $result | Should -BeLikeExactly "docker tag oldname:latest newimage:latest"
         }
 
-        It 'tags docker image with image name with pester tag' {
-            Invoke-DockerTag -SourceImage 'lalaland' -SourceTag 'pester' -TargetImage 'lololand' -TargetTag 'pester'
+        It 'tags image by image name and tag with new image name and tag' {
+            Invoke-DockerTag -ImageName 'oldname' -Tag 'pester' -NewImageName 'newimage' -NewTag 'pester'
```

Comment:
Shouldn't this be 'OldTag' to follow the naming scheme?

Reply:
Yes, that thought crossed my mind. But to stay consistent with the rest of the cmdlets, i chose to reuse the same variable names for original variable type.